htdocs/public/payment/newpayment.php

```diff
@@ -1698,10 +1758,7 @@ class="stripe-button"
 	    }
 	    </style>';
 
-		print '
-
-	    <br>
-	    <form action="'.$_SERVER['REQUEST_URI'].'" method="POST" id="payment-form">';
+		//print '<br><form action="'.$_SERVER['REQUEST_URI'].'" method="POST" id="payment-form">';
```

Comment:
If you remove the "form" here, the button of type "submit" will not work anymore. For example when clicking on "Pay using paypal" or "Pay using paybox".

Reply:
payment intent process payment in javascript in backend so it's more secure and submit form only if  it can be pay (no need redirection for 3D secure or error). I will try to restore it for paypal & paybox